newm-server/build.gradle.kts

```diff
@@ -124,3 +126,32 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
         jvmTarget = "17"
     }
 }
+
+sourceSets {
+    create("integTest") {
+        kotlin.srcDir("src/integTest/kotlin")
+        resources {
+            include("*.conf")
+        }
+        compileClasspath += sourceSets.main.get().output + configurations.testRuntimeClasspath
+        runtimeClasspath += sourceSets.main.get().output + compileClasspath
+    }
+}
+
+configurations["integTestRuntimeOnly"].extendsFrom(configurations.runtimeOnly.get())
+
+task<Test>("integTestGarage") {
```

Comment:
Listing each variation of the tasks seems redundant. I was considering making a custom task, but thought that might be too much.  Open to suggestions.